<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadingInfo</class>
 <widget class="QWidget" name="DownloadingInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="HeaderCardWidget" name="HeaderCardWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>正在下载</string>
   </property>
   <widget class="SimpleCardWidget" name="SimpleCardWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>611</width>
      <height>81</height>
     </rect>
    </property>
    <widget class="StrongBodyLabel" name="StrongBodyLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>总进度</string>
     </property>
    </widget>
    <widget class="ProgressBar" name="DownloadingProgress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>221</width>
       <height>4</height>
      </rect>
     </property>
    </widget>
    <widget class="StrongBodyLabel" name="DownloadingPercent">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Nah%</string>
     </property>
    </widget>
    <widget class="ToggleButton" name="StartAndPause">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>40</y>
       <width>112</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>暂停</string>
     </property>
    </widget>
    <widget class="PushButton" name="Shutdown">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>102</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>终止</string>
     </property>
    </widget>
   </widget>
   <widget class="SimpleCardWidget" name="SimpleCardWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>611</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="StrongBodyLabel" name="StrongBodyLabel_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>检查单</string>
     </property>
    </widget>
    <widget class="ListWidget" name="CheckList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>571</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToggleButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimpleCardWidget</class>
   <extends>CardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HeaderCardWidget</class>
   <extends>SimpleCardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ProgressBar</class>
   <extends>QProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
